app/models/user_option.rb

```diff
@@ -28,6 +28,12 @@ def self.like_notification_frequency_type
     @like_notification_frequency_type ||= Enum.new(always: 0, first_time_and_daily: 1, first_time: 2, never: 3)
   end
 
+  def self.font_sizes
+    @font_sizes = Enum.new(normal: 0, larger: 1, largest: 2)
+  end
+
+  validates :font_size_id, inclusion: { in: UserOption.font_sizes.values }
```

Comment:
I don't think we should name this `font_size_id` as this usually means it `belongs_to :font_size`. Perhaps we can just name this column `font_size`? This will be similar to how we do it for posts where have `Post.types` and `Post#post_type`.

Reply:
I was following the same convention as other enums in this same user_options table. e.g. `homepage_id`

I think having inconsistencies within the same table could be even more confusing than the `belongs_to` confusion 🤔